asset/encoding.go

```diff
@@ -11,11 +11,18 @@ import (
 	"github.com/btcsuite/btcd/btcec/v2"
 	"github.com/btcsuite/btcd/btcec/v2/schnorr"
 	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/txscript"
 	"github.com/btcsuite/btcd/wire"
 	"github.com/lightninglabs/taproot-assets/mssmt"
 	"github.com/lightningnetwork/lnd/tlv"
 )
 
+// TLV types for TapLeaf encode/decode.
```

Comment:
This isn't an asset specific encoding right? Instead, we'll use it in the db, but stashing it here to avoid possible import cycles?

Reply:
Exactly, this seemed like a type that would cause import issues.